notebooks/404-style-transfer-webcam/404-style-transfer.ipynb

```diff
@@ -0,0 +1,464 @@
+    "\n",
+    "> **NOTE**: To use this notebook with a webcam, you need to run the notebook on a computer with a webcam. If you run the notebook on a server, the webcam will not work. However, you can still do inference on a video."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import collections\n",
+    "import sys\n",
+    "import time\n",
+    "\n",
+    "import cv2\n",
+    "import numpy as np\n",
+    "from pathlib import Path\n",
+    "from IPython.display import Markdown as md\n",
+    "from IPython import display\n",
+    "from ipywidgets import interactive, ToggleButtons\n",
+    "from openvino.runtime import Core\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils\n",
+    "from notebook_utils import download_file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Select one of the styles: Mosaic, Rain Princess, Candy, Udnie and Pointilism."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def print_choices(option):\n",
+    "    print(option)\n",
+    "    \"\"\"\n",
+    "    Select the preferred style to transferred\n",
+    "    \"\"\"\n",
+    "    \n",
+    "choice1Select = ToggleButtons(\n",
+    "                    options=['MOSAIC', 'RAIN-PRINCESS', 'CANDY', 'UDNIE', 'POINTILISM'],\n",
+    "                    description=\"Click on the Style:\",\n",
+    "                    disabled=False,\n",
+    "                    style= {'description_width': '300px'}\n",
+    "                    )\n",
+    "   \n",
+    "interactive(print_choices, option=choice1Select)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Model\n",
+    "\n",
+    "### Download the Model\n",
+    "\n",
+    "Model for the selected style will be downloaded to `base_model_dir` if they have not been downloaded before. The selected model comes from the ONNX Model Zoo, which means it must be converted into OpenVINO Intermediate Representation (OpenVINO IR).\n",
+    "\n",
+    "You can also use `\"fast-neural-style-mosaic-onnx\"` as a model name, and the system automatically downloads the model using `omz_downloader`, which is a command-line tool from the `openvino-dev` package."
```

Comment:
You no longer use omz_downloader

Reply:
I added this description as an additional option. Should I remove this?